### PR TITLE
MB-12100 services counselor move history

### PR DESCRIPTION
**Pull request by PDickeyTruss**
## Summary

This PR adds 2 tabs to the top of the Services Counselor Move Info page allowing services counselors to access the  paginated Move History. The 'Move Details' tab displays a numeric tag indicating the count of any unapproved/empty shipping fields (similar to what occurs on this tab for the TIO/TOO).   Also included are a number of related test additions/updates. 

## Video of change

https://user-images.githubusercontent.com/62263329/163472104-c9293169-bb3d-4d83-9070-a9fe54f90aeb.mov

### CODE REVIEW

**Review by YanZ777 — APPROVED** [review]
Aside from my non blocking comment, everything looks good to me!

File: src/pages/Office/ServicesCounselingMoveInfo/ServicesCounselingMoveInfo.jsx [hunk 1]
```diff
@@ -20,8 +22,11 @@ const ServicesCounselingEditShipmentDetails = lazy(() =>
   import('pages/Office/ServicesCounselingEditShipmentDetails/ServicesCounselingEditShipmentDetails'),
 );
 const CustomerInfo = lazy(() => import('pages/Office/CustomerInfo/CustomerInfo'));
+const MoveHistory = lazy(() => import('pages/Office/MoveHistory/MoveHistory'));
 
 const ServicesCounselingMoveInfo = () => {
+  const [unapprovedShipmentCount, setUnapprovedShipmentCount] = React.useState(0);
+
   const [infoSavedAlert, setInfoSavedAlert] = useState(null);
```
Comment (donaldthai) [inline src/pages/Office/ServicesCounselingMoveInfo/ServicesCounselingMoveInfo.jsx:30]: nit (non-blocking): Looks like the `useState` hook is already imported so no need to add the `React.`

**Review by donaldthai — APPROVED** [review]
👍

File: src/components/Office/ServicesCounselingTabNav/ServicesCounselingTabNav.test.jsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  unapprovedShipmentCount: 0,
+  moveCode: 'TESTCO',
+};
+
+describe('Move details tag rendering', () => {
+  it('should render the move details tab container without a tag', () => {
+    render(<ServicesCounselingTabNav {...basicNavProps} />, { wrapper: MemoryRouter });
+
+    const moveDetailsTab = screen.getByTestId('MoveDetails-Tab');
+    expect(within(moveDetailsTab).queryByTestId('tag')).toBeFalsy();
```
[the remark this answers is not in the document]
Comment (PDickeyTruss) [reply src/components/Office/ServicesCounselingTabNav/ServicesCounselingTabNav.test.jsx]: Good spot! I like how much more readable `not.toBeInTheDocument()` is. Fixed ✅

File: src/pages/Office/ServicesCounselingMoveInfo/ServicesCounselingMoveInfo.jsx (hunk 1 above)
Comment (PDickeyTruss) [reply src/pages/Office/ServicesCounselingMoveInfo/ServicesCounselingMoveInfo.jsx]: Good eye, fixed ✅

**Review by cosburn — APPROVED** [review]

**Pull request merged**